http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SamAuthProvider.scala

```diff
@@ -194,6 +194,32 @@ class SamAuthProvider[F[_]: OpenTelemetryMetrics](
     } yield roles.contains(SamRole.Owner)
   }
 
+  def filterWorkspaceReader(
```

Comment:
My thinking here is that as long as there is a policy set for that user, they must be at least a reader

Reply:
have you verified this? ie that its empty when a user has no access to a resource?

Reply:
That is a very good point, I was surprised to see that the unit tests are basically swapping in a completely different logic for that function (aka user being contained in the allowlist). 
This is based on my manual testing (ongoing, setting things up on my BEE now), but if you have an idea to properly test this as a unit test lmk!